server/src/ClientToDBAPI/ClinicalDetailsRoute/mainRoute.ts

```diff
@@ -36,18 +42,35 @@ clinicalDetailsRoute.post('/saveClinicalDetails', (request: Request, response: R
         investigatedPatientId: clinicalDetails.investigatedPatientId,
         isIsolationProblem: clinicalDetails.isIsolationProblem,
         isIsolationProblemMoreInfo: clinicalDetails.isIsolationProblemMoreInfo,
-        isolationAddress: +clinicalDetails.isolationAddress,
         isolationEndTime: clinicalDetails.isolationEndDate,
         isolationStartTime: clinicalDetails.isolationStartDate,
         symptomsStartTime: clinicalDetails.symptomsStartDate,
     }
+
+    const requestAddress: Address = {
+        city: isolationAddress.city,
+        street: isolationAddress.street,
+        floor: +isolationAddress.floor,
+        houseNum: +isolationAddress.houseNum,
+    }
     
-    graphqlRequest(CREATE_INVESTIGATION, { investigation: {
-            ...requestInvestigation
-        }}).then(() => {
+    graphqlRequest(CREATE_ISOLATION_ADDRESS, {
```

Comment:
How do you connect between the isolationAddress and the patient?

Reply:
fixed